513.xml

```diff
@@ -0,0 +1,339 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Copyright Ericsson 2025
```

Comment:
@ejrp minor change, but if this object is registered under the OMA range, the copyright should be assigned to Open Mobile Alliance.
```suggestion
Copyright Open Mobile Alliance 2025
```